ocaml/typing/typeopt.ml

```diff
@@ -425,10 +424,10 @@ and value_kind_variant env ~loc ~visited ~depth ~num_nodes_visited
           List.fold_left_map
             (fun num_nodes_visited (ty, _) ->
                let num_nodes_visited = num_nodes_visited + 1 in
-               (* CR layouts v5: when we add other layouts, we'll need to check
+               (* CR layouts v5: when we add other jkinds, we'll need to check
```

Comment:
I think these comments are really about layouts.  (Though probably there are a lot of such comments, and it's not worth fixing them all)

Reply:
Yeah -- I actually tend to agree with you here. I've looked for uses of jkind in this file and changed as I saw appropriate.